src/js/index.js

```diff
@@ -206,6 +206,12 @@ export const hpe = deepFreeze({
     },
   },
   accordion: {
+    heading: {
+      margin: { vertical: '24px', left: '6px', right: '6px', },
```

Comment:
Can these be done with t-shirt sizes?

Reply:
good point and ill do horizontal 6px not both left and right